### ISSUE TITLE
Running `serve` task fails due to implicit dependency on clojure.tools.nrepl

### ISSUE THREAD

**Issue by pandeiro** [pandeiro/boot-http#63]
```
boot.user=> (require '[pandeiro.boot-http :refer [serve]])
nil
boot.user=> (boot (serve))
java.io.FileNotFoundException: Could not locate clojure/tools/nrepl/server__init.class or clojure/tools/nrepl/server.clj on classpath.
clojure.lang.Compiler$CompilerException: java.io.FileNotFoundException: Could not locate clojure/tools/nrepl/server__init.class or clojure/tools/nrepl/server.clj on classpath., compiling:(boot/repl_server.clj:1:1)
```

**Comment by pandeiro** [comment]
Don't recall seeing this prior to `0.7.6`

**Comment by edannenberg** [comment]
This seems to be introduced with `0.7.6`, see #57 and the related PR #61. Reverted to `0.7.5` for now.

**Comment by pandeiro** [comment]
Fixed as of `0.8.1`
